example/properties_directory/non-scalar/hubbard_parameters.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "hubbard_parameters",
```

Comment:
Should we extract hubbard_u and hubbard_v separately as two different properties? Do we ever need to store both together? 

And what about J?

Reply:
hubbard_u, hubbard_v and hubbard_v_nn (add description full name - nearest neighbors)

Reply:
For reference, currently hp.x does not support calculating Hubbard parameters based on DFT+U+J.